common/src/main/java/com/wynntils/screens/maps/SeaskipperDepartureBoardScreen.java

```diff
@@ -0,0 +1,414 @@
+
+        renderHeight = Texture.DESTINATION_LIST.height() * currentTextureScale;
+
+        mapHeight = renderHeight - renderedBorderYOffset * 2f;
+
+        centerZ = renderY + renderedBorderYOffset + mapHeight / 2f;
+    }
+
+    @Override
+    public void doRender(PoseStack poseStack, int mouseX, int mouseY, float partialTick) {
+        RenderSystem.setShaderColor(1f, 1f, 1f, 1f);
+
+        RenderSystem.enableDepthTest();
+
+        renderMap(poseStack);
+
+        RenderUtils.enableScissor(
+                (int) (renderX + renderedBorderXOffset), (int) (renderY + renderedBorderYOffset), (int) mapWidth, (int)
+                        mapHeight);
+
+        renderDestinations(
+                poseStack,
+                BoundingBox.centered(mapCenterX, mapCenterZ, width / currentZoom, height / currentZoom),
+                1,
+                mouseX,
+                mouseY);
+
+        renderCursor(
+                poseStack,
+                1.5f,
+                Managers.Feature.getFeatureInstance(CustomSeaskipperScreenFeature.class)
+                        .pointerColor
+                        .get(),
+                Managers.Feature.getFeatureInstance(CustomSeaskipperScreenFeature.class)
+                        .pointerType
+                        .get());
+
+        RenderUtils.disableScissor();
+
+        renderBackground(poseStack);
+
+        renderCoordinates(poseStack, mouseX, mouseY);
+
+        RenderUtils.drawScalingTexturedRect(
+                poseStack,
+                Texture.DESTINATION_LIST.resource(),
+                5,
+                destinationListY,
+                0,
+                destinationListWidth,
+                Texture.DESTINATION_LIST.height() * currentTextureScale,
+                Texture.DESTINATION_LIST.width(),
+                Texture.DESTINATION_LIST.height());
+
+        renderWidgets(poseStack, mouseX, mouseY, partialTick);
+    }
+
+    @Override
+    protected void renderMap(PoseStack poseStack) {
+        RenderUtils.enableScissor(
+                (int) (renderX + renderedBorderXOffset), (int) (renderY + renderedBorderYOffset), (int) mapWidth, (int)
+                        mapHeight);
+
+        // Background black void color
+        RenderUtils.drawRect(poseStack, CommonColors.BLACK, renderX, renderY, 0, mapWidth, mapHeight);
+
+        BoundingBox textureBoundingBox =
+                BoundingBox.centered(mapCenterX, mapCenterZ, width / currentZoom, height / currentZoom);
+
+        List<MapTexture> maps = Models.Map.getMapsForBoundingBox(textureBoundingBox);
+        for (MapTexture map : maps) {
+            float textureX = map.getTextureXPosition(mapCenterX);
+            float textureZ = map.getTextureZPosition(mapCenterZ);
+
+            MapRenderer.renderMapQuad(
+                    map, poseStack, centerX, centerZ, textureX, textureZ, mapWidth, mapHeight, 1f / currentZoom);
+        }
+
+        RenderUtils.disableScissor();
+    }
```

Comment:
Did you change anything here or why was this overridden? (I can't look at source right now but this feels really familiar)